tests/PHPStan/Reflection/Annotations/AnnotationsMethodsClassReflectionExtensionTest.php

```diff
@@ -0,0 +1,155 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Reflection\Annotations;
+
+use PHPStan\Broker\Broker;
+use PHPStan\Type\CommonUnionType;
+use PHPStan\Type\IntegerType;
+use PHPStan\Type\ObjectType;
+use PHPStan\Type\VoidType;
+
+class AnnotationsMethodsClassReflectionExtensionTest extends \PHPStan\TestCase
+{
+
+	public function dataMethods(): array
+	{
+		return [
+			[
+				\AnnotationsMethods\Foo::class,
+				[
+					'getInteger' => [
+						'class' => \AnnotationsMethods\Foo::class,
+						'returnType' => new IntegerType(false),
```

Comment:
Please rework asserting those types to $type->describe() which is more future-proof - classes of types can change but descriptions won't. This will be benefitial really soon because I'll be reworking the nullables to be constructed by unions (not by ->isNullable on any type).